imports/ui/explorer-components/unit-btn.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import FloatingActionButton from 'material-ui/FloatingActionButton'
+import FontIcon from 'material-ui/FontIcon'
+import { push } from 'react-router-redux'
+
+export function UnitBtn ({dispatch}) {
```

Comment:
Since you shouldn't pass `dispatch` in, I think UnitBtn will be too simplistic to be its own component named "UnitBtn". You might want to rename it to "AddFAB" (**F**loating **A**ction **B**utton) and introduce it anywhere we already have a similar button, but sounds like a bit of an overkill to me, so I think it's ok to go without it completely